core/sched.c

```diff
@@ -89,11 +91,14 @@ int __attribute__((used)) sched_run(void)
      * since the threading should not be started before at least the idle thread was started.
      */
     int nextrq = bitarithm_lsb(runqueue_bitcache);
-    thread_t *next_thread = container_of(sched_runqueues[nextrq].next->next, thread_t, rq_entry);
+    thread_t *next_thread = container_of(sched_runqueues[nextrq].next->next,
+                                         thread_t, rq_entry);
 
-    DEBUG("sched_run: active thread: %" PRIkernel_pid ", next thread: %" PRIkernel_pid "\n",
-          (kernel_pid_t)((active_thread == NULL) ? KERNEL_PID_UNDEF : active_thread->pid),
-          next_thread->pid);
+    DEBUG(
+        "sched_run: active thread: %" PRIkernel_pid ", next thread: %" PRIkernel_pid "\n",
+        (kernel_pid_t)((active_thread ==
+                        NULL) ? KERNEL_PID_UNDEF : active_thread->pid),
+        next_thread->pid);
```

Comment:
This looks pretty messed up. How about:

```C
    DEBUG(
        "sched_run: active thread: %" PRIkernel_pid ", next thread: %" PRIkernel_pid "\n",
        (kernel_pid_t)((active_thread == NULL)
                      ? KERNEL_PID_UNDEF
                      : active_thread->pid),
        next_thread->pid);
```

This should also be accepted by `uncrustify`